package.json

```diff
@@ -12,7 +12,7 @@
     "build": "vite build",
     "lint": "eslint src test",
     "format": "prettier --write \"{src,test}/**/*.{css,js,json}\"",
-    "prepare": "husky install"
+    "prepare": "npx simple-git-hooks"
```

Comment:
Husky and simple-git-hooks are pretty comparable but the former requires an extra config directory in the project root, which I'm not a huge fan of. Though a switch was fitting.